ReadExcelFile.py

```diff
@@ -19,6 +19,10 @@
 print(my_val)
 
 #Read in values of cells C1 through C5 and print
+#Define the first value to be used as the base of the for loop.
+Total_Cell_Values = 0
+
 for i in range(1,6):
-    C_Cell = sheet.cell(i,3)
-    print(C_Cell.value)
+    C_Cell_value = sheet.cell(i,3).value
+    Total_Cell_Values = Total_Cell_Values + C_Cell_value
+print(Total_Cell_Values)
```

Comment:
Minor stylistic note: I prefer a blank line to separate the body of the loop from the following statement. It's similar to how the "Next" statement in VBA (or a `}` in C) does the separation, but since Python doesn't have those, a blank line does the same thing.